Corona-Warn-App/src/main/java/de/rki/coronawarnapp/http/WebRequestBuilder.kt

```diff
@@ -262,4 +266,79 @@ object WebRequestBuilder {
             }
         }
     }
+
+    private class RegistrationTokenRequestErrorListener<T>(
+        private val requestID: UUID,
+        private val cont: Continuation<T>
+    ) :
+        Response.ErrorListener {
+        override fun onErrorResponse(error: VolleyError?) {
+            val resultingException = when (error) {
+                is TimeoutError -> error
+                null -> WebRequestException(
+                    "an error occurred during a webrequest",
+                    NullPointerException("The volley exception was null")
+                )
```

Comment:
Can we somehow prevent repeating this?

Reply:
I am working on the HTTP Layer and this maybe will be reworked together with every other request :)